controllers/suite_test.go

```diff
@@ -48,12 +49,11 @@ var k8sManager ctrl.Manager
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	RunSpecsWithDefaultAndCustomReporters(t,
```

Comment:
Has this API been deprecated? Or changed to include the reporter(NewlineReporter) by default?

Reply:
yes its depricated, by default it provides junit report